### PR TITLE
Add slider

### PR DESCRIPTION
**Pull request by rjrishika**

### CODE REVIEW

**Comment by rjrishika** [comment]
@vedant-jain03 please review my pr

File: src/components/Homepage.jsx [hunk 1]
```diff
@@ -257,33 +257,40 @@ function Homepage() {
...
+                                  <th scope="col"><img src={template4} alt="" /></th>
+                                </tr>
+                                <tr className={`templates ${template==='template5'?"active":""}`} onClick={()=>settemplate("template5")}>
+                                  <th scope="col"><img src={template5} alt="" /></th>
+                                </tr>
+                                <tr className={`templates ${template==='template6'?"active":""}`} onClick={()=>settemplate("template6")}>
+                                  <th scope="col"><img src={template6} alt="" /></th>
+                                </tr>
+                                <tr className={`templates ${template==='template7'?"active":""}`} onClick={()=>settemplate("template7")}>
+                                  <th scope="col"><img src={template7} alt="" /></th>
```
Comment (vedant-jain03) [inline src/components/Homepage.jsx]: This is not needed, you can just add this css:
```
.maincontainer .leftmost {
    flex: 0.20 1;
    padding: 1rem;
    display: block;
    align-items: center;
    flex-direction: column;
    color: white;
    border-right: 1px gray solid;
    text-align: center;
    overflow-y: scroll;
}
```

Remove this and please test the code before raising PR!

Thankyou, let me know if you have any doubts

**Review by vedant-jain03 — CHANGES_REQUESTED** [review]
Thanks @rjrishika kindly check my reviews and make the PR update accordingly!

**Comment by rjrishika** [comment]
changes done

File: src/components/homepage.css [hunk 2]
```diff
@@ -256,4 +258,12 @@
...
\ No newline at end of file
+}
+.my-custom-scrollbar {
+  position: relative;
+  height: auto;
+  overflow: auto;
+  }
+  .table-wrapper-scroll-y {
+  display: block;
+  }
```
Comment (vedant-jain03) [inline src/components/homepage.css]: Is this still needed? If not, kindly remove it!

**Review by vedant-jain03 — CHANGES_REQUESTED** [review]

File: src/components/homepage.css (hunk 2 above)
Comment (vedant-jain03) [reply src/components/homepage.css]: Other LGTM, we can merge the PR after this change!

**Review by vedant-jain03 — APPROVED** [review]
@rjrishika Great work, thanks!!!

**Pull request merged**